### DISCUSSION TITLE
Timeout / Lockout

### DISCUSSION THREAD

**Discussion by Colebiagetti** [guysoft/FullPageOS#551] [Q&A]
Hello, currently using fullpage OS to display a browser on boot. I find that it is timing out after 10 minutes. Is there a specific file/folder in the OS that I can change it so that it never times out?

**Comment by Colebiagetti** [comment]
?

**Reply by guysoft** [reply]
Hey, the screen should remain on with FullPageOS by default.

What version of FullPageOS are you using? And what display? Have you tried another display?

Here is the line that disables power management:
https://github.com/guysoft/CustomPiOS/blob/devel/src/modules/gui/filesystem/opt/scripts/start_gui#L14

**Comment by jmarocchi** [comment]
I had to reinstall today and am running into the same issue. Even though DISABLE_POWER_MANAGEMENT=yes, I still am seeing the timeout. Raspberry Pi 3b running the latest. 

I think I found a solution. If you go into rasp-config -> Display Options -> Screen Blanking and make sure it's disabled. Will report back if I run into the issue again

**Comment by g13ryj** [comment]
I am having exactly the same issue here - using the latest stable build.  Also tried the nightly.  Screen Blanking disabled too yet it still goes off after about 10 minutes.  Any ideas?
